### ISSUE TITLE
Proposal to add support for GIFT in pyslet

### ISSUE THREAD

**Issue by anna-liao** [swl10/pyslet#59]
@swl10 I would be interested to add support for GIFT in pyslet.  Perhaps based on this [GIFT to QTI converter](https://github.com/csev/gift2qti), written in Python.  Do you think this feature addition would be appropriate for pyslet?  If yes, my plan is to implement this new feature and submit it as a pull request. Thanks!

**Comment by swl10** [comment]
Yes, it seems appropriate.  I feel like it is worth reaching out to @csev too who may be interested in reviewing the work as something of a Python fan himself.
